pkg/controller/networkpolicy/antreanetworkpolicy.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package networkpolicy
+
+import (
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/apis/networking"
+	secv1alpha1 "github.com/vmware-tanzu/antrea/pkg/apis/security/v1alpha1"
+	antreatypes "github.com/vmware-tanzu/antrea/pkg/controller/types"
+)
+
+// addANP receives Antrea NetworkPolicy ADD events and creates resources
```

Comment:
substitue Antrea NetworkPolicy with AppNetworkPolicy in this file.. comments/log messages func names etc

Reply:
Done